src/ui/ui_insert_email.py

```diff
@@ -75,7 +75,7 @@ def setupUi(self, DialogInsertEmail):
 
         self.lineEdit_password = QLineEdit(self.gridLayoutWidget)
         self.lineEdit_password.setObjectName(u"lineEdit_password")
```

Comment:
Why using u""? Is this automatically generated code?

Reply:
ui_ python files under ui folder are autogenerated by qtdesigner. I don't touche them as they get overwritten anytime i change UI. we can skip such files from review..